sdk/appconfiguration/test-resources.json

```diff
@@ -72,7 +72,8 @@
                 "name": "[parameters('sku')]"
             },
             "properties": {
-                "endpoint": "[variables('endpointValue')]"
+                "endpoint": "[variables('endpointValue')]",
+                "disableLocalAuth": true
```

Comment:
@weshaggard thanks for your generic "add disablelocalauth" advice it definitely unblocked me here.

@mrm9084 this setting will have to be set to enable `new-testresources appconfiguration`.

Reply:
@scbedd, the test still seems to be failing. What is the step I need to take?

Reply:
Two things need to happen.

- The disable local auth actually breaks 8 tests. So those will need to be fixed after that change + re-record
- The varying different versions of `azure-appconfiguration` are where the recording mismatches (which include `label`) are happening. You're getting mismatches because the different versions of `azure-appconfiguration` either include or DON'T include this `label=blah` value. You can either raise the minimum supported version of `azure-appconfiguration`, or add a sanitizer for the breaking query value.

Reply:
Here is the mismatch:

```
    request <https://Sanitized.azconfig.io/kv/message?label=%00&api-version=1970-01-01>
    record  <https://Sanitized.azconfig.io/kv/message?api-version=1970-01-01>
```

I believe `request` is using newest `appconfiguration` in the above example.